skunkworks_crow/src/main/java/org/odk/share/services/SenderService.java

```diff
@@ -80,22 +80,20 @@ public void startUploading(long[] instancesToSend, int port, int mode) {
         }
     }
 
-    private void startJob(JobRequest request) {
-        request.schedule();
-        Timber.d("Starting upload job %d : ", request.getJobId());
+    private void startJob(WorkRequest request) {
+        workManager.enqueue(request);
+        Timber.d("Starting upload job %s : ", request.getId().toString());
         currentJob = request;
     }
 
     public void cancel() {
         if (currentJob != null) {
-            Job job = JobManager.instance().getJob(currentJob.getJobId());
-            if (job != null) {
-                job.cancel();
+                workManager.cancelWorkById(currentJob.getId());
                 rxEventBus.post(new UploadEvent(UploadEvent.Status.CANCELLED));
             } else {
                 Timber.e("Pending job not found : %s", currentJob);
             }
             currentJob = null;
         }
     }
```

Comment:
Also, as this is closing braces please indent this as well.

Reply:
removed the indentation for the SenderService closing braces.